manifests/repo.pp

```diff
@@ -0,0 +1,12 @@
+# Configure the foreman repo
+class foreman::repo(
+) {
+  if ! $::foreman::custom_repo {
+    foreman::repos { 'foreman':
+      repo     => $::foreman::repo,
+      gpgcheck => $::foreman::gpgcheck,
+    }
+  }
+
+  class { '::foreman::repos::extra': }
```

Comment:
Maybe it is good to pass all parameters for the foreman-cli use case, but I'm leaning to pulling in ```::foreman::repos::extra``` in here.